lib/mix/lib/mix/scm.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Mix.SCM do
   @moduledoc """
   This module provides helper functions and defines the
-  behaviour required by any SCM used by mix.
+  behaviour required by any SCM used by `Mix`.
```

Comment:
No backticks I think. We are referring to Mix as the tool, right? And not the module `Mix`.